gym/core.py

```diff
@@ -58,12 +69,6 @@ def _render(self, mode='human', close=False):
             return
         raise NotImplementedError
 
-    # Will be automatically set when creating an environment via
```

Comment:
maybe worth keeping this comment in __new__ so it's easier to trace the magic that sets spec